benchmark/utils/metrics/estimation.py

```diff
@@ -0,0 +1,22 @@
+import cv2 as cv
+
+from .base_metric import BaseMetric
+from ..factory import METRICS
+
+@METRICS.register
+class Estimation(BaseMetric):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def forward(self, model, *args, **kwargs):
+        img, label = args
+
+        self._timer.reset()
+        for _ in range(self._warmup):
+            model.infer(img, label)
+        for _ in range(self._repeat):
+            self._timer.start()
+            model.infer(img, label)
+            self._timer.stop()
+
+        return self._getResult()
```

Comment:
I dont see any fundamental differences between recognition.py and this file. Can we reuse `Recognition` for metric? Maybe we should rename `Recognition` to a proper name that makes sense for other models.

Reply:
Of course we can, but I don't know which name is appropriate.